server.js

```diff
@@ -3,14 +3,32 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
+
 const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/happyThoughts"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
+const Thought = mongoose.model('Thought', {
+  message: {
+    type: String,
+    required: true,
+    minlenght: 5,
+    maxlength: 140
+  },
+  hearts: {
+    type: Number,
+    default: 0,
+    required: true
```

Comment:
If you defining the default field, there is no need to make it required, because it will never be empty :)

Reply:
True, will fix!